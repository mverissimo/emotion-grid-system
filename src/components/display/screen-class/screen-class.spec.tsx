import { render } from '@testing-library/react';

import { ScreenClass } from '.';

describe('ScreenClass', () => {
  it('should render with current screen %s', () => {
    const { container } = render(
      <ScreenClass
        render={(screen) => {
          return <div>{screen}</div>;
        }}
      />,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
