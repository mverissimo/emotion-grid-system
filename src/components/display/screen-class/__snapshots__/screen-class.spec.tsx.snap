// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ScreenClass > should render with current screen %s 1`] = `
<div>
  initial
</div>
`;
